README.md

```diff
@@ -69,3 +69,16 @@ Run the tests
 ```
 ./vendor/bin/phpunit
 ```
+
+## Use Docker for testing
+We provide a Dockerfile with Ubuntu 16.04 and PHP7
+Start testing via the following:
+```
+docker build -t sofortlib .
+docker run -ti -v $(pwd):/app sofortlib bash -c "composer install && vendor/bin/phpunit"
```

Comment:
You could run it like that to have all tests run, correct. But think about someone who'd like to somehow "interact" with the code... you run unit tests by building the container, that should be quite sufficient to run it afterwards and edit files.
But by mounting your files into the container, you could simply use it e.g. for development reasons